Directory.Build.props

```diff
@@ -274,7 +274,8 @@
     <PackageProjectUrl>https://dot.net</PackageProjectUrl>
     <Owners>microsoft,dotnetframework</Owners>
     <IncludeSymbols>true</IncludeSymbols>
-    <RuntimeIdGraphDefinitionFile>$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'Microsoft.NETCore.Platforms', 'src', 'runtime.json'))</RuntimeIdGraphDefinitionFile>
+    <SourceRuntimeIdentifierGraphFile>$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'Microsoft.NETCore.Platforms', 'src', 'runtime.json'))</SourceRuntimeIdentifierGraphFile>
+    <GeneratedRuntimeIdentifierGraphFile>$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'Microsoft.NETCore.Platforms', '$(Configuration)', 'runtime.json'))</GeneratedRuntimeIdentifierGraphFile>
```

Comment:
Is the generated runtime identifier graph file configuration aware? If not, we should put this into the `BaseOutputPath` which is configuration agnostic.

Reply:
We now have three properties to define the runtime identifier file, meaning the same thing:
- SourceRuntimeIdentifierGraphFile: Use the runtime identifier file that is checked-in.
- GeneratedRuntimeIdentifierGraphFile: Use the generate runtime identifier file.
- BundledRuntimeIdentifierGraphFile (in livebuild.targets): Use the `SourceRuntimeIdentifierGraphFile`.

Instead, it would be great if we could only have one: `BundledRuntimeIdentifierGraphFile` which is the publicly exposed SDK property:

(in Directory.Build.props)
```xml
<BundledRuntimeIdentifierGraphFile>$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'Microsoft.NETCore.Platforms', 'runtime.json'))<BundledRuntimeIdentifierGraphFile>
<BundledRuntimeIdentifierGraphFile Condition="!Exists('$(BundledRuntimeIdentifierGraphFile)')">$([MSBuild]::NormalizePath('$(LibrariesProjectRoot)', 'Microsoft.NETCore.Platforms', 'src', 'runtime.json'))</BundledRuntimeIdentifierGraphFile>
```

and remove the property from livebuild.targets and update Platforms.csproj.

Reply:
I had to keep `GeneratedRuntimeIdentifierGraphFile` to pass it to the `Platforms.csproj`. I did merge `SourceRuntimeIdentifierGraphFile` and `BundledRuntimeIdentifierGraphFile`.

Reply:
`GeneratedRuntimeIdentifierGraphFile` is only relevant to a single project: `Platforms.csproj`. It shouldn't be shared across all projects. Consider using `$(BaseOutputPath)runtime.json` instead in the two places in `Platforms.csproj` instead.